src/components/DateScrollPicker/DateScrollPicker.tsx

```
@@ -1,55 +1,40 @@
 /** @jsxImportSource @emotion/react */
-import {useTheme} from '@theme/DesignProvider';
-import {useCallback, useState} from 'react';
+import {useCallback, useEffect, useState} from 'react';
```

ESLint

[error] 'useCallback' is defined but never used.

(@typescript-eslint/no-unused-vars)